Formula/cython-mode.rb

```diff
@@ -0,0 +1,34 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class CythonMode < EmacsFormula
+  desc "Emacs major mode for Cython"
+  homepage "http://cython.org"
+  url "https://github.com/cython/cython/archive/0.24.tar.gz"
+  sha256 "b60b91f1ec88921a423d5f0a5e2a7c232cdff12d9130088014bf89d542ce137b"
+  head "https://github.com/cython/cython.git"
+
+  depends_on :emacs
+
+  def install
+    cd "Tools"
+    byte_compile "cython-mode.el"
+    (share/"emacs/site-lisp/cython-mode").install "cython-mode.el",
+                                                  "cython-mode.elc"
+  end
+
+  def caveats; <<-EOS.undent
+    Add the following to your init file:
```

Comment:
I've stopped included caveats that aren't Homebrew-specific, since they can get out of date and end up at best duplicating upstream docs.